uswds/uswds
# USWDS: File Input: Remove aria disabled file input. Resolves #5144

## Summary
Remove `aria-disabled` attribute from `usa-file-input` element to allow screen readers to read out label tag.

## Breaking change
This is **not** a breaking change


## Related issue
Closes #5144 

## Preview link
Preview link:


## Problem statement
Currently, when File Input is initialized, the `aria-disabled` attribute is added to the parent element. This causes the label to no longer be read by screen readers, which goes against best practices for screen readers.

## Solution
Remove JS which sets this attribute. Now the File Input is still read as disabled, but the label is also ready by screenreaders

## Testing and review
1. Visit the File Input Disabled story.
2. Activiate screen reader
3. Check that the label is read
4. Check that File Input component is read as disabled.
